### DISCUSSION TITLE
Are there events for dialog other than dismissed?

### DISCUSSION THREAD

**Discussion by mtanco** [h2oai/wave#1291] [Q&A]
I'm not finding in docs or the code based a list of events for the dialog, or how to use events. I see there is a default `dismissed` event, but can do you do others?

For example, I would like to close this dialog after the user clicks the link on this page: 
![image](https://user-images.githubusercontent.com/4793561/157712911-750be4af-5306-4ea5-b66b-79f928333738.png)

**Comment by mturoci** [comment]
`dismissed` is the only event on dialog currently. The only way I can see this work is to:

* create a button
*  handle button click
*  within btn handler: close the dialog and start download via 
```py
q.page['meta'] = ui.meta_card(box='', script=ui.inline_script('window.open(<your_download_url>)'))
```

**Reply by mtanco** [reply]
Thank you for the the inline java script! Good idea. 

I'll open a separate issues to make the dismissed events docs string less vague :D
